docker-storage-setup.sh

```diff
@@ -720,6 +719,21 @@ unit: sectors
 
 ${dev}1 : start=     2048, size=  ${size}, Id=8e
 EOF
+}
+
+create_partition_parted(){
+  local dev="$1"
+  parted $dev --script mklabel dos mkpart primary 0% 100%
```

Comment:
May be we should use `0` as start value which means start at 0 MB and use `-1s` as end value to tell parted to create partition till end of disk.

I think it will still do the right alignment and create partition  beginning at 2048 sector.